tdp/cli/commands/restart_required.py

```diff
@@ -89,9 +89,17 @@ def restart_required(
 
         operation_runner = OperationRunner(dag, ansible_executor, service_managers)
 
-        deployment = operation_runner.run_nodes(
+        config_deployment = operation_runner.run_nodes(
```

Comment:
In my opinion, you should have only one `run_nodes` with a correct filter:

The filter you need should be something like `node_filter=re.compile(r".+_(config|start)")`

(import re package to use regular expression)